### ISSUE TITLE
ReferenceError: ts is not defined on no-indexed-access-on-enums rule

### ISSUE THREAD

**Issue by sdedieu** [criteo/eslint-plugin-criteo#30]
### Description

The rule no-indexed-access-on-enums rule gives the following error when linter is executed: 
```
ReferenceError: ts is not defined 
Occurred while linting C:\mb\0\frontend\cds-15\cds-library\projects\cds-library\src\lib\components\date-period-picker\date-period-picker-selection\date-period-picker-selection.service.ts:69
Rule: "criteo/no-indexed-access-on-enums" at MemberExpression (C:\mb\0\frontend\cds-15\node_modules\eslint-plugin-criteo\lib\rules\no-indexed-access-on-enums.js:37:13)
```

### Environment
- Node 16.20.2
- Angular 15.2.10
- Eslint 8.52.0
- eslint-plugin-criteo 4.12.0

### Current Behaviour
The following code causes a true positive:
```
const options: number[] = [1,2,3];
for (let i = 0; i < options.length; i++) {
  const option = options[i]; // trigger the no-indexed-access-on-enums rule
}
```

### Expected Behaviour
The upper code should not trigger the `no-indexed-access-on-enums` rule since `options` is an array and not an enum.

**Comment by XavierDupessey** [comment]
cc @adamperea @ConnorUllmann

**Comment by XavierDupessey** [comment]
@sdedieu You can downgrade to v4.11.1 until a fix, v4.12.0 only introduces this new rule

**Comment by XavierDupessey** [comment]
The rule is disabled in v4.13.0
